content/en/01-about-pixie/06-troubleshooting.md

```diff
@@ -135,3 +141,55 @@ Run the `px/tracepoint_status` script. It should show a longer error message in
 *How do I remove a tracepoint table?*
 
 It is not currently possible to remove a table. Instead, we recommend renaming the table name (e.g. table_name_0) while debugging the script.
+
+## Troubleshooting a Pixie Plugin
+
+### Why isn't my data exporting when using the Pixie Plugin?
+
+Here are some things to check if you are not seeing exported data when using the Pixie Plugin:
+
+- Is Pixie working properly? If you just installed Pixie, [did it finish deploying](#troubleshooting-deployment-how-do-i-check-the-status-of-pixie's-components)? Are you able to see data in the [Live UI](/using-pixie/using-live-ui/)?
+
+- Did you [enable the plugin](/reference/plugins/plugin-system/#enabling-a-plugin) from within the Live UI Admin page?
+
+- When you enabled the plugin, did you use the correct endpoint? [Check the logs](#troubleshooting-a-pixie-plugin-how-do-i-check-the-pixie-logs-for-plugin-errors).
+
+- When you enabled a third party Pixie Plugin, did you use the correct type of API key? [Check the logs](#troubleshooting-a-pixie-plugin-how-do-i-check-the-pixie-logs-for-plugin-errors).
+
+- Did you [configure the data retention scripts](/reference/plugins/plugin-system/#long-term-data-retention)? You will need to enable one or more preset scripts and/or custom export scripts.
+
+- If you [added a custom export script](/reference/plugins/plugin-system/#long-term-data-retention-creating-custom-export-scripts), did you first test the script in the Live UI's scratch pad? See the [Export OpenTelemetry Data](/tutorials/integrations/otel/) tutorial for directions.
+
+- Do you see errors in the `kelvin-*` pod logs?
+
+- Do you see errors in the `vizier-query-broker-*` pod logs?
+
+### How do I check the Pixie logs for plugin errors?
+
+1. To debug Pixie Plugin issues, first check for errors in the `kelvin-*` pod logs.
+
+> If you're following along with the [Export OpenTelemetry Data](/tutorials/integrations/otel/) tutorial and your OpenTelemetry collector has not started up yet, you'll see the following error:
+
+```bash
+Query c6139bff-880d-473a-98aa-2a6ca9543dd3 failed, reason: Internal: OTel export (carnot node_id=55) failed with error 'UNAVAILABLE'. Details: DNS resolution failed for service: otel-collector.default.svc.cluster.local:4317
+```
+
+> If you provided an incorrect `custom export URL` (when [enabling the plugin](/reference/plugins/plugin-system/#enabling-a-plugin)), you may see an error like the following:
+
+```bash
+Query c6139bff-880d-473a-98aa-2a6ca9543dd3 failed, reason: Internal : OTel export (carnot node_id=157) failed with error 'UNAVAILABLE'. Details: failed to connect to all addresses
+```
```

Comment:
Wasn't clear. That first error is not just because of the tutorial. I'd bucket it as a similar error to the `custom export URL` you've listed. 

IMO would just do 
> If you provided an incorrect ...
```
Query c6139bff-880d-473a-98aa-2a6ca9543dd3 failed, reason: Internal : OTel export (carnot node_id=157) failed with error 'UNAVAILABLE'. Details: failed to connect to all addresses

# or 

Query c6139bff-880d-473a-98aa-2a6ca9543dd3 failed, reason: Internal: OTel export (carnot node_id=55) failed with error 'UNAVAILABLE'. Details: DNS resolution failed for service: otel-collector.default.svc.cluster.local:4317
```